components/match2/commons/match_group_input.lua

```diff
@@ -399,4 +399,26 @@ function MatchGroupInput.getCommonTournamentVars(obj)
 	return obj
 end
 
+function MatchGroupInput.readMvp(match)
+	if not match.mvp then return end
+	local mvppoints = match.mvppoints or 1
+
+	-- Split the input
+	local players = mw.text.split(match.mvp, ',')
+
+	-- parse the players to get their information
+	local parsedPlayers = Table.mapValues(players, function(player)
+		local link = mw.ext.TeamLiquidIntegration.resolve_redirect(mw.text.split(player, '|')[1]):gsub(' ', '_')
+		for _, opponent in Table.iter.pairsByPrefix(match, 'opponent') do
+			for _, lookUpPlayer in pairs(opponent.match2players) do
+				if link == lookUpPlayer.name then
```

Comment:
Is this tested on a player with lowercase first letter?

Reply:
should we add a fallback in case it not finding the link in match2players?
something like:
```lua
	local parsedPlayers = Table.mapValues(players, function(player)
		local link = mw.ext.TeamLiquidIntegration.resolve_redirect(mw.text.split(player, '|')[1]):gsub(' ', '_')
		for _, opponent in Table.iter.pairsByPrefix(match, 'opponent') do
			for _, lookUpPlayer in pairs(opponent.match2players) do
				if link == lookUpPlayer.name then
					return Table.merge(lookUpPlayer, {team = opponent.name, template = opponent.template})
				end
			end
		end
		-- the following 2 lines are new
		local playerParts = mw.text.split(player, '|')
		return {name = playerParts[1], displayname = playerParts[#playerParts ]}
	end)
```

Reply:
Nah, I think it's good as is. We need to make ensure properly setup pages anyhow for the app